tests/pages/services/ServiceActions-cy.js

```diff
@@ -127,14 +130,13 @@ describe('Service Actions', function () {
             url: /marathon\/v2\/apps\/\/sleep/,
             response: {
               message: {message: 'Not Authorized to perform this action!'}
-            },
-            delay: 100
```

Comment:
We should leave this delay here. It's part of the test.
Basically, the button should be briefly disabled and we check that bellow. Then we check that the class is removed.